org-asterion/__main__.py

```diff
@@ -79,78 +82,155 @@ def org_exists(self):
 asterion_infra_aws_prod = aws.organizations.OrganizationalUnit("asterion-infra-aws-prod", parent_id=asterion_infra_aws.id)
 
 # Output asterion environment ou id's
-pulumi.export("asterion-infra-aws OU ID", asterion_infra_aws.id)
-pulumi.export("Dev OU ID", asterion_infra_aws_dev.id)
-pulumi.export("Test OU ID", asterion_infra_aws_test.id)
-pulumi.export("Prod OU ID", asterion_infra_aws_prod.id)
-
-# Create an asterion group for the users
-admin_group = aws.iam.Group(
-    "admins",
-    path="/users/"
-    )
+pulumi.export("asterion dev ou id", asterion_infra_aws.id)
+pulumi.export("asterion dev ou id", asterion_infra_aws_dev.id)
+pulumi.export("asterion test ou id", asterion_infra_aws_test.id)
+pulumi.export("asterion prod ou id", asterion_infra_aws_prod.id)
+
+#####################################################################################
+
+#####################################################################################
```

Comment:
These blocks are perhaps a sign that this `__main__.py` file should be split and call sub `.py` files.

Reply:
Agreed! I can split these up in the next PR, if all good - as per your previous convo thread on this PR.

Reply:
Agreed with @shawngerrard  we would implement these improvements in a future pull request, resolving.